Formula/unisonlang.rb

```diff
@@ -0,0 +1,49 @@
+require "language/node"
+
+class Unisonlang < Formula
+  desc "Friendly programming language from the future"
+  homepage "https://unison-lang.org/"
+  url "https://github.com/unisonweb/unison.git",
+      tag:      "release/M3",
+      revision: "97cce8d23147df30b0f80ae745968db09f4e1a44"
+  version "1.0.0-M3"
+  license "MIT"
+  head "https://github.com/unisonweb/unison.git", branch: "trunk"
+
+  depends_on "haskell-stack" => :build
+  depends_on "node" => :build
+
+  on_linux do
+    depends_on "xz" => :build
+  end
+
+  resource "unison-ui" do
+    url "https://github.com/unisonweb/codebase-ui.git",
+        tag:      "release/M3",
+        revision: "05df3a235dec7d4a03ff1c588a32eb6daca288d4"
+  end
+
+  def install
+    # Build the ucm binary
+    system "stack", "build", "--flag", "unison-parser-typechecker:optimized"
+
+    # Install the ucm binary
+    local_install_root = Pathname.new `stack path | awk '/local-install-root/{print $2}'`.to_s.strip
+    cp local_install_root/"bin/unison", prefix/"ucm"
+    bin.install_symlink prefix/"ucm"
```

Comment:
Why not just install it in one go?

Reply:
There are other files in the bin/unison/ build directory that must live beside the binary, but that can't go in `#{prefix}/bin/` because they aren't binaries